cmd/ltagent/init.go

```diff
@@ -91,12 +85,12 @@ func RunInitCmdF(cmd *cobra.Command, args []string) error {
 	if ok, err := isInitDone(config.ConnectionConfiguration.ServerURL, userPrefix); err != nil {
 		return err
 	} else if ok {
-		mlog.Warn("init already done")
+		log.Warn("init already done")
```

Comment:
Unrelated but this log was never showing for me because we would exit right away. I added a defer above to flush the logger so it gets showed.